packages/freezed/lib/src/freezed_generator.dart

```diff
@@ -545,6 +557,15 @@ extension on Element {
       throwOnUnresolved: false,
     );
   }
+
+  bool get hasGenericArgumentFactories {
+    if (!hasJsonSerializable) return false;
+
+    final annotation =
+        const TypeChecker.fromRuntime(JsonSerializable).firstAnnotationOf(this);
+    final value = annotation.getField('genericArgumentFactories').toBoolValue();
```

Comment:
missing null check? firstAnnotationOf can return null

Reply:
The line before checks if it does not have the `JsonSerializable` annotation and returns false. Should we still check if `firstAnnotationOf` returns null?